File: src/SIL.XForge.Scripture/Models/VerseRefData.cs
using SIL.Scripture;

namespace SIL.XForge.Scripture.Models
{
    public class VerseRefData
    {
        public int BookNum { get; set; }
        public int ChapterNum { get; set; }
        public int VerseNum { get; set; }
        public ScrVersType Versification { get; set; }
    }
}

File: src/SIL.XForge.Scripture/ClientApp/src/app/core/models/verse-ref-data.ts
import { ScrVersType } from './sfdomain-model.generated';

export class VerseRefData {
  constructor(
    public bookNum?: number,
    public chapterNum?: number,
    public verseNum?: number,
    public versification: ScrVersType = ScrVersType.English
  ) {}
}
